src/helpers/DataRequirementHelpers.ts

```diff
@@ -445,3 +446,40 @@ function hasMeasurementPeriodInfo(options: CalculationOptions, effectivePeriod?:
     options.measurementPeriodStart || options.measurementPeriodEnd || effectivePeriod?.start || effectivePeriod?.end
   );
 }
+
+/**
+ * Get a flattened list of all related artifacts in the measure bundle.
+ *
+ * @param measureBundle The measure bundle to fetch all RelatedArtifacts from
+ * @returns List of flattened related artifacts.
+ */
+export function getFlattenedRelatedArtifacts(measureBundle: fhir4.Bundle): fhir4.RelatedArtifact[] {
+  const measure = MeasureBundleHelpers.extractMeasureFromBundle(measureBundle);
+  // add the measure itself
+  const relatedArtifacts: fhir4.RelatedArtifact[] = [
+    {
+      type: 'depends-on',
+      display: 'Measure',
```

Comment:
Maybe here we can try to use the Measure's `name` in the display if it is defined? It should be defined in most cases

```suggestion
      display: `Measure ${measure.name ?? ''}`,
```